sovrin_client/test/cli/test_nym.py

```diff
@@ -255,7 +255,7 @@ def testNewKeyChangesWalletsDefaultId(be, do, poolNodesStarted, poolTxnData,
        expect=["Nym {} added".format(idr)])
 
 
-def testSend2NymsSucceedsWhenBatched(
+def test_send_same_nyms_fails_when_batched(
```

Comment:
The second request in the batch shouldn't be written (the same behavior as when not batched).  The test verifies this by making sure the first request is what is written.  I've renamed the test to clarify this.